### PR TITLE
move the stores from global stores directory to provider store directory

### PR DESCRIPTION
**Pull request by raymondz1**
## What this PR does / why we need it

1. move stores to each individual provider. 
2. 
## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/store/Azure.store.mc.tsx [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-import { StoreDispatch } from '../../shared/types/types';
-import azureReducer from '../reducers/Wizard.reducer';
-import { STORE_SECTION_FORM } from '../reducers/Form.reducer';
-import { AzureCloud } from '../../shared/constants/App.constants';
-import { AZURE_DEFAULT_VALUES } from '../../shared/constants/defaults/azure.defaults';
-import { AZURE_FIELDS } from '../../views/management-cluster/azure/azure-mc-basic/AzureManagementClusterBasic.constants';
+import { StoreDispatch } from '../../../../shared/types/types';
+import azureReducer from '../../../../state-management/reducers/Wizard.reducer';
+import { STORE_SECTION_FORM } from '../../../../state-management/reducers/Form.reducer';
+import { AzureCloud } from '../../../../shared/constants/App.constants';
```
Comment (swalner-vmware) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/store/Azure.store.mc.tsx:8]: nit: alpha order

**Review by swalner-vmware — APPROVED** [review]
Nice overall; just the alpha nits

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx [hunk 2]
```diff
@@ -3,15 +3,14 @@ import React, { useContext, useState, useEffect } from 'react';
 
 // Library imports
 import { CdsButton } from '@cds/react/button';
-import { CdsIcon } from '@cds/react/icon';
 import { FormProvider, SubmitHandler, useForm } from 'react-hook-form';
 import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
 import * as yup from 'yup';
 
 // App imports
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx:11]: nit: remove extra line break

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-basic/AzureBasicReviewStep.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { useNavigate } from 'react-router-dom';
 import { CdsButton } from '@cds/react/button';
 import { CdsIcon } from '@cds/react/icon';
 // App imports
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-basic/AzureBasicReviewStep.tsx:7]: pls add a line break between 6 & 7

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/store/Aws.store.mc.tsx [hunk 4]
```diff
@@ -1,12 +1,12 @@
 // React imports
 import React, { createContext, ReactNode, useReducer } from 'react';
 // App imports
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/store/Aws.store.mc.tsx:3]: please add line break between 2 & 3

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-basic/AwsBasicReviewStep.tsx [hunk 5]
```diff
@@ -6,7 +6,7 @@ import { CdsButton } from '@cds/react/button';
 import { CdsIcon } from '@cds/react/icon';
 // App imports
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-basic/AwsBasicReviewStep.tsx:7]: please add line break between 6 & 7

Comment (raymondz1) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-basic/AwsBasicReviewStep.tsx:9]: added

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-basic/AzureBasicReviewStep.tsx (hunk 3 above)
Comment (raymondz1) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-basic/AzureBasicReviewStep.tsx:9]: added

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx (hunk 2 above)
Comment (raymondz1) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx]: removed

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/store/Aws.store.mc.tsx (hunk 4 above)
Comment (raymondz1) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/store/Aws.store.mc.tsx:4]: added

**Pull request merged**